epilepsy12/tests/general_functions_tests/test_cohort_calculations.py

```diff
@@ -5,16 +5,12 @@
 # Third party imports
 
 # RCPCH imports
-from epilepsy12.general_functions import (
-    cohort_number_from_first_paediatric_assessment_date,
-)
 
+# def test_cohort_number_from_enrolment_date_returns_correct_num(e12_case_factory):
+#     """
+#     Test that cohort_number_from_enrolment_date() returns correct num, given different Date of First Paediatric Assessment (DoFPAs).
 
-def test_cohort_number_from_enrolment_date_returns_correct_num(e12_case_factory):
```

Comment:
This test still passes for me on this branch?

Reply:
don't think it is testing anything here..